setup.cfg

```diff
@@ -51,19 +51,22 @@ rfi_file_monitor.operations =
     DropboxUploader = rfi_file_monitor.operations.dropbox_uploader:DropboxUploaderOperation
     S3Downloader = rfi_file_monitor.operations.s3_downloader:S3DownloaderOperation
     S3Copier = rfi_file_monitor.operations.s3_copier:S3CopierOperation
-    UrlDownloader = rfi_file_monitor.operations.url_downloader:UrlDownloaderOperation
     DirectoryCompressor = rfi_file_monitor.operations.directory_compressor:DirectoryCompressorOperation
 rfi_file_monitor.engines =
     FileWatchdogEngine = rfi_file_monitor.engines.file_watchdog_engine:FileWatchdogEngine
     TemporaryFileEngine = rfi_file_monitor.engines.temporary_file_engine:TemporaryFileEngine
     AWSS3BucketEngine = rfi_file_monitor.engines.aws_s3_bucket_engine:AWSS3BucketEngine
-    URLsEngine = rfi_file_monitor.engines.urls_engine:URLsEngine
     DirectoryWatchdogEngine = rfi_file_monitor.engines.directory_watchdog_engine:DirectoryWatchdogEngine
     CephS3BucketEngine = rfi_file_monitor.engines.ceph_s3_bucket_engine:CephS3BucketEngine
 rfi_file_monitor.preferences =
     AllowedFilePatternsPreference = rfi_file_monitor.preferences:AllowedFilePatternsPreference
     IgnoredFilePatternsPreference = rfi_file_monitor.preferences:IgnoredFilePatternsPreference
-gui_scripts =
+rfi_file_monitor.files =
+    RegularFile = rfi_file_monitor.files.regular_file:RegularFile
+    WeightedRegualrFile = rfi_file_monitor.files.regular_file:WeightedRegularFile
+    S3Object = rfi_file_monitor.files.s3_object:S3Object
+    Directory = rfi_file_monitor.files.directory:Directory
+console_scripts =
```

Comment:
I think you want to keep `gui_scripts` around here. Probably introduced after debugging.

Reply:
hi Tom thanks for pointing this out. This is currently WIP - and will be the first of a few rather large PRs going in. I will write a test or something so it does not get merged.